### PR TITLE
Update packages version

### PR DESCRIPTION
**Pull request by lisez**
- Update all dependencies.
- Use `@deno/astro-adapter` instead `deno-astro-adapter`.
- Update scripts that use astro commands instead.
- Use `astro.config.ts` instead.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -8,22 +8,22 @@
       "name": "deno-astro-template",
       "version": "1.0.0",
       "dependencies": {
-        "@astrojs/deno": "^4.2.0",
-        "astro": "^2.8.3",
-        "deno-astro-adapter": "^4.2.0"
+        "@astrojs/deno": "^5.0.1",
```
Comment (kt3k) [inline package-lock.json:11]: Is this used now?

File: package.json [hunk 2]
```diff
@@ -3,21 +3,20 @@
...
-    "preview": "deno run -A --unstable ./dist/server/entry.mjs",
-    "astro": "astro",
+    "preview": "deno run -A ./dist/server/entry.mjs",
     "format": "deno fmt && prettier --write ."
   },
   "dependencies": {
-    "@astrojs/deno": "^4.2.0",
-    "astro": "^2.8.3",
-    "deno-astro-adapter": "^4.2.0"
+    "@astrojs/deno": "^5.0.1",
```
Comment (kt3k) [inline package.json:13]: Is this used now?

Comment (lisez) [reply package.json]: oops, I missed it.

**Review by kt3k — APPROVED** [review]
Thank you for the updates!

**Pull request merged**